src/erlcloud_sns.erl

```diff
@@ -422,6 +423,18 @@ publish_to_target(TargetArn, Message, Subject, Config) ->
 publish_to_target(TargetArn, Message, Subject, AccessKeyID, SecretAccessKey) ->
     publish_to_target(TargetArn, Message, Subject, new_config(AccessKeyID, SecretAccessKey)).
 
+-spec publish_to_phone(string(), sns_message()) -> string().
+publish_to_phone(TargetArn, Message) ->
```

Comment:
please write couple line of doc reffering AWS doc http://docs.aws.amazon.com/sns/latest/api/API_Publish.html and mentioning that TargetArn can be also just a phone number string.